Airport Ticket Booking System Tests/FlightValidationTests/FlightValidationShould.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Airport_Ticket_Booking_System.Entities.Flights.Core;
+using Airport_Ticket_Booking_System.Utilities;
+using FluentAssertions;
+using Moq;
+
+namespace Airport_Ticket_Booking_System_Tests.FlightValidationTests
+{
+    public class FlightValidationShould
+    {
+        private Flight flight;
+
+        public FlightValidationShould()
+        {
+            flight = new Flight();
+        }
+
+        [Fact]
+        public void ValidatePrice_WhenPricesAreValid_ShouldReturnTrue()
+        {
+            flight.EconomyPrice = 500;
+            flight.BusinessPrice = 800;
+            flight.FirstClassPrice = 1300;
+
+            var result = FlightValidation.ValidatePrice(flight, out string priceErrors);
+
+            result.Should().BeTrue();
+            priceErrors.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void ValidatePrice_WhenAnyPriceIsInvalid_ShouldReturnFalseAndProvideErrorMessages()
+        {
+            flight.EconomyPrice = 0;  
+            flight.BusinessPrice = -20;  
+            flight.FirstClassPrice = -1;
+
+            var result = FlightValidation.ValidatePrice(flight, out string priceErrors);
+
+            result.Should().BeFalse();
+            priceErrors.Should().Contain(ErrorMessages.EconomyPriceInvalid)
+                                 .And.Contain(ErrorMessages.BusinessPriceInvalid)
+                                 .And.Contain(ErrorMessages.FirstClassPriceInvalid);
+        }
```

Comment:
try to have one test method for each prop
and have the test cases as theorydata on inlinedata